### ISSUE TITLE
Numbro's syntax not working on numericFormat with typescript

### ISSUE THREAD

**Issue by JoseMCO** [handsontable/handsontable#7420]
### Description
Can't seem to be able to use Numbro's syntax for numeric type cells while using typescript. As mentioned [here](https://github.com/handsontable/handsontable/issues/5645) and [here](https://forum.handsontable.com/t/gh-5645-numeric-format-pattern-issue-whole-units-display-as-0-00-and-not-0/2656/19) an update was made to use the `pattern: { mantissa: 0 }` syntax for the numericFormat option, but typescript won't compile it because of a type mismatch (it expects a string for the `pattern` option and not an object)

```
const columns = [
    { data: 'cxp', name: 'CxP', type: 'numeric', numericFormat: { pattern: { mantissa: 0 }}, readOnly: true },
];

const data = [
    { cxp: 12.5 },
    { cxp: 18.502 },
    { cxp: 234 },
    ...
];

return (
    <HotTable
      ref={ref}
      licenseKey={HANDSONTABLE_KEY}
      language="es-MX"
      columns={columns}
      data={data}
    />
);
```

### Environment
* React wrapper version: 4.0.0
* Handsontable version: 8.2.0
* Browser Name and version: any
* Operating System: any

**Comment by wojciechczerniak** [comment]
@JoseMCO No need, transfered already.

Note for @handsontable/devs: We can import and use: https://github.com/BenjaminVanRyseghem/numbro/blob/78538abbc3711c2f5603ae596a90875b790b9be3/numbro.d.ts#L67-L97 as we did with Pikaday options: https://github.com/handsontable/handsontable/blob/34793d5e9ee10176941ebe2589996daa80b5effd/handsontable.d.ts#L1

**Comment by aninde** [comment]
Fixed by v8.4.0
Tested locally with v8.4.0.

**Comment by AMBudnik** [comment]
Thank you for testing and fixing!

@JoseMCO I hope that It will also work for you.

Please update to [v8.4.0](https://github.com/handsontable/handsontable/issues/7998) and let us know if it does.
